torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -400,13 +402,13 @@ c10::optional<ReductionParams> scheduleReduction(
   // 1.) I am only binding Tensor Dimension sizes.  I am not binding scalar
   // values. 2.) I am only binding the IterDomain.extent().  Do I need to worry
   // about the start?
```

Comment:
Can you move https://github.com/csarofeen/pytorch/blob/20_7_6_devel/torch/csrc/jit/codegen/cuda/executor.cpp#L81-L113 to a utility function in https://github.com/csarofeen/pytorch/blob/20_7_6_devel/torch/csrc/jit/codegen/cuda/executor_utils.h and use that?